docs/getting-started/configure-via-openshift-ui.md

```diff
@@ -0,0 +1,59 @@
+## Configuring VDO via Openshift Web Console
+
+The easiest way to configure VDO is to configure the CRD's via Openshift Web Console, follow the steps below for the configuration.
+
+#### Pre-requisite
+
+You should have below data in handy.
+
+- IP address of vcenter
+- Secure Connection - If you choose to establish a secure connection to vcenter, you need to provide a ssl thumbprint
+- Login credentials for vcenter
+- Datacenter(s) - you can provide a comma separated list of datacenters
```

Comment:
-  datacenters in which the kubernetes nodes are present. This is required by CPI and CSI to manage the cluster